src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -448,6 +472,14 @@ private Response abortRun(RepairRun repairRun, RepairUnit repairUnit, int segmen
     return Response.ok().entity(new RepairRunStatus(newRun, repairUnit, segmentsRepaired)).build();
   }
 
+  private Response editRun(
+          RepairRun repairRun, RepairUnit repairUnit, int segmentsRepaired, Double intensity
+  ) throws ReaperException {
```

Comment:
unusual line wrapping of method declaration.

Reply:
Linter throwed error about line size. Suggestions?

Reply:
fair enough.
the project's precedence on this style would be the following
```
  private Response editRun(
          RepairRun repairRun,
          RepairUnit repairUnit,
          int segmentsRepaired,
          Double intensity) throws ReaperException {
```

Reply:
Okay.